### PR TITLE
RHBA-524: Rename Library to Projects

### PR DESCRIPTION
**Pull request by jomarko**
This is ensemble together with:
- #1572 
- https://github.com/kiegroup/kie-wb-distributions/pull/728

### CODE REVIEW

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
@paulovmr would you mind have a look please?

**Comment by jomarko** [comment]
jenkins please retest this

**Comment by jomarko** [comment]
jenkins execute full downstream build

**Comment by jomarko** [comment]
downstream build seems to be successful

File: kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-api/src/main/java/org/kie/workbench/common/screens/library/api/preferences/LibraryPreferences.java [hunk 1]
```diff
@@ -30,6 +31,9 @@
     @Property(bundleKey = "LibraryPreferences.ProjectPreferences")
     LibraryProjectPreferences projectPreferences;
 
+    @Property(bundleKey = "LibraryPreferences.AdvancedGavPreferences")
```
Comment (paulovmr) [inline kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-api/src/main/java/org/kie/workbench/common/screens/library/api/preferences/LibraryPreferences.java]: @jomarko Before, the .niogit/system/preferences.git directory structure had this form:

```
.
├── config
│   └── global
│       └── global
│           ├── FileExport.preferences
│           ├── GAVPreferences.preferences
│           └── LibraryPreferences.preferences
└── readme.md
```

Now it has this (which shouldn't have changed):

```
.
├── config
│   └── global
│       └── global
│           ├── FileExport.preferences
│           └── LibraryPreferences.preferences
└── readme.md
```

To fix this (keep the GAVPreferences in a different file), could you try to add the property `shared = true` to the `@Property` annotation and check the files structure and content?

Comment (paulovmr) [reply kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-api/src/main/java/org/kie/workbench/common/screens/library/api/preferences/LibraryPreferences.java]: Also, please test running the workbench with a preferences.git created without your changes, and with modified values for the preferences. Then, when you run it with your changes, the saved values should be loaded the same.

Comment (jomarko) [reply kie-wb-common-screens/kie-wb-common-library/kie-wb-common-library-api/src/main/java/org/kie/workbench/common/screens/library/api/preferences/LibraryPreferences.java]: @paulovmr thanks for review, will do.

**Comment by jomarko** [comment]
@paulovmr thanks one more time for review, I checked
- if structure in `.niogit/system/preferences.git` is same before and after my changes
- if preferences form older version of workbench are loaded correctly
- after reload admin page and restart workbench, if preferences loaded as saved before

Please let me know if there is anything else I should improve in PR

**Comment by jomarko** [comment]
jenkins please retest this

**Comment by jomarko** [comment]
@manstis this is ready for merge.

**Pull request merged**